pkg/controller/controller.go

```diff
@@ -314,46 +325,63 @@ func (c *RolloutController) adjustStatefulSetsGroupReplicas(ctx context.Context,
 	// are up-to-date on the models. If we modify the StatefulSet, they no longer are.
```

Comment:
Above, the function comment says:

> The method returns "true" early if the number of replicas in any StatefulSet was adjusted.

Maybe we wanted to say "only if the number..."?